pumpkin-lib/src/propagators/cumulative/time_table/time_table_over_interval_incremental.rs

```diff
@@ -466,6 +681,165 @@ mod insertion {
     }
 }
 
+/// Contains the functions necessary for removing the appropriate profiles into the time-table
+/// based on the reduced mandatory part.
+mod removal {
```

Comment:
Given that there is no `use super::*` here, perhaps this can be moved to a separate file?